### ISSUE TITLE
Governance document is masculine coded

### ISSUE THREAD

**Issue by rabernat** [pangeo-data/governance#11]
I am transferring this comment by @mcgibbon from #5 (which is now closed) to a new issue so we don't lose track of it.

>As it is currently written, [the section on actions of the steering board] is [strongly masculine-coded](http://gender-decoder.katmatfield.com/results/e1531905-1a00-4e2d-a116-9282897691f4). It risks putting women off applying, but will probably encourage men to apply ([paper](http://gender-decoder.katmatfield.com/static/documents/Gaucher-Friesen-Kay-JPSP-Gendered-Wording-in-Job-ads.pdf)). We should avoid coding the description of the steering committee to be masculine-biased in pursuit of diversity as is being discussed in #8. If you like I can make particular suggestions on changes to correct this, or @niallrobinson (or someone else) could make some changes focusing on masculine-coded words, phrases, and concepts and increasing the emphasis of the collaborative aspect of Pangeo.

**Comment by mcgibbon** [comment]
Thanks! I suppose this comment refers back to #10.

**Comment by jhamman** [comment]
@kmpaul - do you think we can or should rope in UCAR's  Office of Diversity, Equity, and Inclusion to help with some of this?

**Comment by kmpaul** [comment]
Yes. I’ve reached out to them, but they are busy very busy right now. I hope they can offer some help in a week or two.

**Comment by chiaral** [comment]
same here, I reached out to the office here, but she will reply once she has some time to take a look at the documents.

**Comment by kmpaul** [comment]
I just pinged our Office of Diversity, Equity, and Inclusion a second time.  Hopefully they will have some time to help with this soon.

**Comment by rabernat** [comment]
It was resolved by having an open comment period on the draft governance document. During that long time, no one proposed any specific suggestions (in the form of PRs) to the governance document related to this issue. Once the governance document was merged, I closed all outstanding issues on the topic.

If you or anyone else would like to propose a specific change, there is a [policy for how to do that](https://github.com/pangeo-data/governance/blob/master/governance.md#changing-the-governance-documents).

I would be happy to reopen the issue if you wish, to indicate that it is still unresolved.

**Comment by mcgibbon** [comment]
Given the issue can be resolved by a PR at any time (at least, once we've established the Steering Council), and it is unresolved, I'd like to re-open the issue. The resolution could well be that the DEI offices said the document is fine as-is. Do @kmpaul or @chiaral have any updates from their respective DEI offices?

**Comment by kmpaul** [comment]
The comments that I received from the NCAR Office of DEI were noted (and acted upon) in reference to issue #17.

**Comment by mcgibbon** [comment]
Thank you @kmpaul, I was unaware of #17 and #20. I'm preparing a PR to address the first point brought up in #17.

**Comment by kmpaul** [comment]
@mcgibbon Thank you!
